### ISSUE TITLE
Can't find my inverter after install

### ISSUE THREAD

**Issue by lbreggi** [WillCodeForCats/solaredge-modbus-multi#573]
### Describe the bug

Hi! I'm getting an error after the instalation of this integration. it completes the instalation but can't find my inverter, meters or battery.

### Expected behavior

Be able to add my inverter and the other components of the solar system

### Screenshots

![IMG_0074](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/e057e938-b8f3-49f1-bfaf-f5c86fc2350d)
![IMG_0073](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/388a0d4c-7d88-4ed8-8300-ec41c877f598)
![IMG_0072](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/2c0484e8-c2df-4ba6-9611-e70abca063e1)
![IMG_0071](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/36f71d7f-1c60-472b-9187-0e311c3282d3)
![IMG_0070](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/21b61599-a03c-45e5-98eb-3837350be691)
![IMG_0069](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/1777415b-5d7f-48e9-8e3c-ccf2d389535e)
![IMG_0068](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/bf06cb3d-20f9-4161-93b4-e8b1ff25461d)
![IMG_0067](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/25729150/fa237e7e-d167-401d-8a2e-ad06204b3b33)

### Diagnostic File

[home-assistant_solaredge_modbus_multi_2024-04-02T18-41-43.830Z.log](https://github.com/WillCodeForCats/solaredge-modbus-multi/files/14842455/home-assistant_solaredge_modbus_multi_2024-04-02T18-41-43.830Z.log)

### Home Assistant Version

2024.3.3

### solaredge-modbus-multi Version

2.4.13

### Installation Type

HAOS

### Read the Instructions

- [X] I have read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki
- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] This issue is not about a template error (template errors should not be reported as an issue, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

**Comment by WillCodeForCats** [comment]
Your inverter stops responding and is timing out the integration:

```
2024-04-02 13:41:26.298 DEBUG (MainThread) [custom_components.solaredge_modbus_multi.hub] Registers received requested : 40 40
2024-04-02 13:41:38.311 DEBUG (MainThread) [custom_components.solaredge_modbus_multi] Finished fetching SolarEdge Coordinator data in 12.345 seconds (success: False)
```

The long delay between 13:41:26 and 13:41:38 should be where the integration polls the inverter for data.

Try turning off the option "Auto-Detect Additional Entities" under "Configure" button in the integration panel.

**Comment by lbreggi** [comment]
You are the best! now I was able to detect the meters, but can't find the battery. I believe there is a way to extend the time out query? thanks!!

**Comment by WillCodeForCats** [comment]
You have battery turned off.

**Comment by lbreggi** [comment]
yes, the issue is that I'm getting the time out again when I turn the option to look for the battery on

**Comment by WillCodeForCats** [comment]
Unfortunately that's not an integration problem I can fix, if your inverter does not respond to some modbus registers then that might be a firmware issue.

**Comment by lbreggi** [comment]
got it.. anyway thanks for your help! just to share I'm running the latest version of the firmware, just in case if someone else face the problem (there is a screenshot with all the versions at the top). thanks again!

**Comment by WillCodeForCats** [comment]
You should read this discussion where other people have issues with battery related registers not working on some inverter models: [Battery integration #63](https://github.com/WillCodeForCats/solaredge-modbus-multi/discussions/63)

The "Auto-Detect Additional Entities" setting is related to battery registers, so if that doesn't work then it makes sense that batteries won't work either.

**Comment by WillCodeForCats** [comment]
Ref: issue #394

**Comment by WillCodeForCats** [comment]
Ref: PR #416
